sql/src/test/java/org/opensearch/sql/sql/parser/AstQualifiedNameBuilderTest.java

```diff
@@ -32,9 +32,7 @@ public void canBuildRegularIdentifierForSQLStandard() {
   @Test
   public void canBuildRegularIdentifierForOpenSearch() {
     buildFromTableName(".opensearch_dashboards").expectQualifiedName(".opensearch_dashboards");
-    buildFromIdentifier("@timestamp").expectQualifiedName("@timestamp");
     buildFromIdentifier("logs-2020-01").expectQualifiedName("logs-2020-01");
-    buildFromIdentifier("*logs*").expectQualifiedName("*logs*");
```

Comment:
Does
```java
buildFromIdentifier("'*logs*'").expectQualifiedName("*logs*");
```
works?

Reply:
Just tried it. It does not work with the following message 

```
line 1:0 mismatched input ''*logs*'' expecting {'FIRST', 'LAST', 'AVG', 'COUNT', 'MAX', 'MIN', 'SUM', 'FULL', 'DATE', 'DAYOFWEEK', 'TIME', 'TIMESTAMP', 'D', 'T', 'TS', 'FIELD', '.', ID, BACKTICK_QUOTE_ID}

Failed to parse query due to offending symbol ['*logs*'] at: ''*logs*'' <--- HERE... More details: Expecting tokens in {'FIRST', 'LAST', 'AVG', 'COUNT', 'MAX', 'MIN', 'SUM', 'FULL', 'DATE', 'DAYOFWEEK', 'TIME', 'TIMESTAMP', 'D', 'T', 'TS', 'FIELD', '.', ID, BACKTICK_QUOTE_ID}
org.opensearch.sql.common.antlr.SyntaxCheckException: Failed to parse query due to offending symbol ['*logs*'] at: ''*logs*'' <--- HERE... More details: Expecting tokens in {'FIRST', 'LAST', 'AVG', 'COUNT', 'MAX', 'MIN', 'SUM', 'FULL', 'DATE', 'DAYOFWEEK', 'TIME', 'TIMESTAMP', 'D', 'T', 'TS', 'FIELD', '.', ID, BACKTICK_QUOTE_ID}
```

Reply:
what about backticks? 
```
buildFromIdentifier("`*logs*`").expectQualifiedName("*logs*"); 
```

Reply:
I think I tried something similar with Forest yesterday.
Trying to run it also fails, but with this error instead:

```
/Users/gabrielf/Repos/opensearch-project-sql/sql/src/main/java/org/opensearch/sql/sql/parser/AstExpressionBuilder.java:101: error: cannot find symbol
    return visit(ctx.qualifiedName());
                    ^
  symbol:   method qualifiedName()
  location: variable ctx of type TableNameContext
```

Reply:
Disregard my last comment containing the error message. I realized that I was using my parser which I am currently editing to try and implement a different rule for Indexes/Table Names. The following test using backticks does work

```buildFromIdentifier("`*logs*`").expectQualifiedName("*logs*");```

but this one still does not work 

`buildFromIdentifier("'*logs*'").expectQualifiedName("*logs*");`

Reply:
`'*logs*'` is incorrect, because identifiers could be wrapped by backticks `` ` `` only.
So, valid one is `` `*logs*` ``.